packages/app/src/utils/dynamicUI/extractDynamicConfig.ts

```diff
@@ -22,12 +24,22 @@ export type MenuItem =
       };
     };
 
+export type MenuItem = {
+  name: string;
+  title: string;
+  icon: string;
+  children: MenuItem[];
+  priority?: number;
+  to?: string;
```

Comment:
Can to be also an absolute link like `https://support.my-company.com/create-ticket?for=rhdh` ?

Or should we add another property `href` for external links?

External links (that starts with http:// or https://) should be opened in a new tab by default.

Reply:
A menu link item, should only go to a plugin's page. I don't think we need an external link in the menu item. @ShiranHi How do you think?

Reply:
@jerolimov  Actually, this `to` prop does support absolute links, it's just it opens the URL on the current browser tab.